src/databaseScripts/InsertData.sql

```diff
@@ -14,6 +14,225 @@ INSERT INTO band(band_level, band_name)
 	(5, "Associate"),
 	(6, "Apprentice"),
 	(6, "Trainee");
+select * from band;
```

Comment:
Do you need the select in here?

Reply:
You're right, that is not needed, I deleted it.